docs/modules/ROOT/pages/toolbox.adoc

```diff
@@ -4,24 +4,34 @@
 
 Experiment with your Neo4j GraphQL API using the purpose-built web IDE.
 
-> This tool is currently for development purposes only!
-
-== Example
-
-image::toolbox-editor-view.png[title="Query editor view"]
+> This tool is for development and experimentation purposes only!
 
 == Link
 
-Access the Neo4j GraphQL Toolbox here: https://graphql-toolbox.neo4j.io/
+Access the Neo4j GraphQL Toolbox under https://graphql-toolbox.neo4j.io
 
 == Usage
 
-1. Connect to the database with your credentials
+1. Connect to a Neo4j database with valid credentials
 2. Define (or Introspect) the Type definitions
 3. Build the Neo4j GraphQL schema
 4. Experiment, query and play
 
 == Requirements
 
 1. A Neo4j database, see https://neo4j.com/docs/desktop-manual/current/[Neo4j Desktop] or use a https://neo4j.com/cloud/[Neo4j AuraDB]
-2. Make sure the database fulfills the requirements stated xref::introduction.adoc#introduction-requirements[here], including the necessary plugins.
+2. Make sure the Neo4j database fulfills xref::introduction.adoc#introduction-requirements[these requirements], including the necessary plugins
+
+== Example
+
+Define your Type definitions directly in the editor or Introspect the Neo4j database you connected to.
```

Comment:
Is it "Type definitions", or "type definitions"?

Reply:
also "Introspect" or "introspect"?

If it's a verb, I would suggest the lowercase variant.

Reply:
https://github.com/neo4j/graphql/pull/3508/commits/fe971fe9f08ed898b186185ee1167ac0c837ad9a